core/domain/question_domain.py

```diff
@@ -198,30 +226,29 @@ def from_dict(cls, question_dict):
             Question. The corresponding Question domain object.
         """
         question = cls(
-            question_dict['question_id'],
+            question_dict['id'],
             question_dict['question_data'],
             question_dict['question_data_schema_version'],
-            question_dict['language_code'])
+            question_dict['language_code'],
+            question_dict['status'])
 
         return question
 
     @classmethod
-    def create_default_question(
-            cls, question_id, language_code):
+    def create_default_question(cls, question_id):
         """Returns a Question domain object with default values.
 
         Args:
             question_id: str. The unique ID of the question.
-            language_code: str. The ISO 639-1 code for the language this
-                question is written in.
 
         Returns:
             Question. A Question domain object with default values.
         """
         return cls(
             question_id, exp_domain.State.create_default_state(
-                feconf.DEFAULT_INIT_STATE_NAME, is_initial_state=True),
-            feconf.CURRENT_QUESTION_SCHEMA_VERSION, language_code)
+                feconf.DEFAULT_INIT_STATE_NAME, is_initial_state=True
+                ).to_dict(), feconf.CURRENT_QUESTION_SCHEMA_VERSION,
```

Comment:
Deindent by 4 to match opening paren on line 248.

You can also create a new temporary variable for the second arg if doing it inline starts looking too complicated.

Reply:
Done.